packages/meteor-wallet-app/README.md

```diff
@@ -0,0 +1,66 @@
+# @near-wallet-selector/meteor-wallet-app
+
+This is the [Meteor Wallet (for in app only)](https://meteorwallet.app) package for NEAR Wallet Selector.
```

Comment:
I don't understand what it means for it to be `in app only`

Is this for a mobile app? or for a wallet embedded within apps?

If it is a mobile app, you could change it to:

```
This package is for the [Meteor Wallet **mobile app**](https://meteorwallet.app) package for NEAR Wallet Selector.
```